kits/insurance-denial-appeal/prompts/insurance-denial-appeal_analyze_user.md

```
@@ -0,0 +1,31 @@
+## Claim Denial Letter (raw text)
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)